invokeai/app/invocations/mask.py

```diff
@@ -118,3 +119,28 @@ def invoke(self, context: InvocationContext) -> MaskOutput:
             height=mask.shape[1],
             width=mask.shape[2],
         )
+
+
+@invocation(
+    "tensor_mask_to_image",
+    title="Tensor Mask to Image",
+    tags=["mask"],
+    category="mask",
+    version="1.0.0",
+)
+class MaskTensorToImageInvocation(BaseInvocation, WithMetadata, WithBoard):
+    """Convert a mask tensor to an image."""
+
+    mask: TensorField = InputField(description="The mask tensor to convert.")
+
+    def invoke(self, context: InvocationContext) -> ImageOutput:
+        mask = context.tensors.load(self.mask.tensor_name)
+        # Ensure that the mask is binary.
+        if mask.dtype != torch.bool:
+            mask = mask > 0.5
+        mask_np = mask.float().cpu().detach().numpy() * 255
```

Comment:
There's no need for `detach` here. We can also just do `mask_np = (mask.float() * 255).byte().cpu().numpy()` and get rid of the next line. The math and type casting will be ensured by the byte call.

Reply:
[e6a512a](https://github.com/invoke-ai/InvokeAI/pull/6701/commits/e6a512aa867d88558df1dde1162c9f1fe60a49c2)